packages/shared/src/components/feeds/FeedSettings/useFeedSettingsEdit.tsx

```diff
@@ -49,16 +49,31 @@ export const useFeedSettingsEdit = ({
   const { displayToast } = useToastNotification();
   const { showPrompt } = usePrompt();
   const { logEvent } = useLogContext();
+  const { user } = useAuthContext();
 
   const feed = useMemo<Feed>(() => {
+    // calculate main feed from user object as fallback for now
+    // in the future feedList can return main feed as well
```

Comment:
What do we need this for it's the context for the modal/page?

Reply:
my feed is currently not returned in feedList (because we render it manually in sidebar on different spot), but since we know how it looks I just create its object based on user id (instead of finding in feedList), don't want to change on API now since it would break current prod (or not break but show items in sidebar)